cmd/influxd/run.go

```diff
@@ -90,6 +90,9 @@ func Run(config *Config, join, version string, logWriter *os.File) (*messaging.B
 			go a.ListenAndServe()
 		}
 
+		// broker needs the server to occasionally run continuous queries
+		b.RunContinuousQueryLoop(s)
```

Comment:
Still doesn't look right. If the node is only Broker, the continuous query loop will never run. And since the continuous query loop only runs on the Raft leader, it means it would never run if the cluster uses dedicated Raft nodes.